pkg/proofing/resource_definitions.go

```diff
@@ -1,4 +1,5 @@
 package proofing
 
+// ResourceDefinitions
```

Comment:
What is the purpose of this comment?

Reply:
I added some additional docs. This was mostly added just to make the linter happy.